src/pods/canvas/canvas-put-table-on-top.business.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { putTableOnTop } from './canvas.business';
+import { TableVm } from './canvas.vm';
+
+describe('putTabbleOnTop', () => {
+  it('should return empty array, if I have a table and the array is empty', () => {
+    // Arrange
+    const tables: TableVm[] = [];
+    const tableId = 'someTableId';
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result).toEqual([]);
+  });
+
+  it('should return the same array, if table and array with one element,but it is not the same as table', () => {
+    // Arrange
+    const tables: TableVm[] = [
+      { id: '1', fields: [], tableName: 'Table 1', x: 0, y: 0 },
+    ];
+    const tableId = '2';
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result).toEqual(tables);
+  });
+
+  it("should return an array with a single table, which is the same table if it's the only one", () => {
+    // Arrange
+    const TableId = '1';
+    const tables: TableVm = {
+      id: TableId,
+      fields: [],
+      tableName: 'Table 1',
+      x: 0,
+      y: 0,
+    };
+    const arrayWithOnlyTable: TableVm[] = [tables];
+
+    // Act
+    const result = putTableOnTop(TableId, arrayWithOnlyTable);
+
+    // Assert
+    expect(result).toEqual(arrayWithOnlyTable);
+  });
+
+  it("should return array with table on top if it's already in the first position", () => {
+    // Arrange
+    const tableId = '1';
+    const tables: TableVm[] = [
+      { id: tableId, fields: [], tableName: 'Table 1', x: 0, y: 0 },
+      { id: '2', fields: [], tableName: 'Table 2', x: 10, y: 10 },
+    ];
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result[0]).toBe(tables[1]);
```

Comment:
This works, but maybe we can be more exact testing this:
  - Expect that table[1] now is on [0] position.
  - Expect that table[0] is now on [1] position.

Something like: 

```diff
    // Assert
    expect(result[0]).toBe(tables[1]);
    expect(result[1]).toBe(tables[0]);
```

By doing this we are testing that the elements are swapped in this case